src/utils.rs

```diff
@@ -40,28 +40,34 @@ pub fn exec_cmd(cmd: &str, args: &[&str]) -> Option<CommandOutput> {
     };
     match command.as_str() {
         "crystal --version" => Some(CommandOutput {
-            stdout: String::from("Crystal 0.32.1 (2019-12-18)"),
+            stdout: String::from(
+                "\
+Crystal 0.35.1 (2020-06-19)
+
+LLVM: 10.0.0
+Default target: x86_64-apple-macosx\n",
+            ),
             stderr: String::default(),
         }),
         "dummy_command" => Some(CommandOutput {
-            stdout: String::from("stdout ok!"),
-            stderr: String::from("stderr ok!"),
+            stdout: String::from("stdout ok!\n"),
+            stderr: String::from("stderr ok!\n"),
         }),
         "elixir --version" => Some(CommandOutput {
             stdout: String::from(
                 "\
 Erlang/OTP 22 [erts-10.6.4] [source] [64-bit] [smp:8:8] [ds:8:8:10] [async-threads:1] [hipe]
 
-Elixir 1.10 (compiled with Erlang/OTP 22)",
+Elixir 1.10 (compiled with Erlang/OTP 22)\n",
             ),
             stderr: String::default(),
         }),
         "elm --version" => Some(CommandOutput {
-            stdout: String::from("0.19.1"),
+            stdout: String::from("0.19.1\n"),
             stderr: String::default(),
         }),
         "go version" => Some(CommandOutput {
-            stdout: String::from("go version go1.12.1 linux/amd64"),
+            stdout: String::from("go version go1.12.1 linux/amd64\n"),
             stderr: String::default(),
         }),
         "helm version --short --client" => Some(CommandOutput {
```

Comment:
Github doesn't let me add this as a suggestion on the line below but the output from helm includes a new line
```
❯ helm version --short --client | cat -A
v3.2.4+g0ad800e$
```